### PR TITLE
Migrate details pages to use PatternFly composable table

### PR DESCRIPTION
**Pull request by dlabrecq**
Migrating the remaining details pages to use the newer PatternFly composable version. The legacy table will be deprecated for the upcoming PatternFly breaking change release.

https://issues.redhat.com/browse/COST-2349

Screen Shot 2022-11-03 at 3 53 37 PM

### CODE REVIEW

**Review by dlabrecq — COMMENTED** [review]
Updated to remove unused translations and fixed data labels

File: src/locales/messages.ts [hunk 1]
```diff
@@ -1275,7 +1265,7 @@ export default defineMessages({
       'ocp {OpenShift details table} ' +
       'other {}}',
     description: 'Breakdown back to page titles',
-    id: 'breakdownBackToTitles',
+    id: 'dataTableAriaLabel',
```
Comment (gitdallas) [inline src/locales/messages.ts:1268]: Should the description be updated?

**Review by gitdallas — APPROVED** [review]
Looks good. Nice to see some simplification and code reduction in some places. A little nit:

File: src/locales/messages.ts (hunk 1 above)
Comment (dlabrecq) [reply src/locales/messages.ts:1268]: Yes, good catch

**Pull request merged**